testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -188,17 +188,6 @@ List jobConfigs = [
         githubTeamReviewers: ['devops'],
         emails: ['devops@edx.org'],
     ],
-    [
-        org: 'edx',
-        repoName: 'edx-notes-api',
-        defaultBranch: 'master',
-        pythonVersion: '3.8',
-        cronValue: cronOffHoursBusinessWeekday,
-        githubUserReviewers: [],
-        githubTeamReviewers: ['devops'],
```

Comment:
Hmm, looks like we should also update the other repos that mention the `devops` team in this file.